app/controllers/follower_accounts_controller.rb

```diff
@@ -4,6 +4,6 @@ class FollowerAccountsController < ApplicationController
   include AccountControllerConcern
 
   def index
-    @accounts = @account.followers.page(params[:page]).per(FOLLOW_PER_PAGE)
+    @follows = Follow.where(target_account: @account).order(id: :desc).page(params[:page]).per(FOLLOW_PER_PAGE).preload(:account)
```

Comment:
What about moving the query `Follow.where(...)` into an `ordered_follows` method or something, and then just have `@follows = ordered_follows.page(...)` in the action.